src/ILCompiler/src/RemovingILProvider.cs

```diff
@@ -179,5 +216,6 @@ public enum RemovedFeature
         Etw = 0x1,
         FrameworkResources = 0x2,
         Globalization = 0x4,
+        Comparers = 0x8,
```

Comment:
Nit: This may be called `OptimizedComparers` to better describe what it does.

Reply:
Naming is always hard, but the concern I have about the word Optimized is that this is also slightly about correctness. The correctness aspect is not big (.NET Native shipped without this feature for a while before someone hit it), but it is there.